qemu/tcg/tcg.c

```diff
@@ -2375,7 +2375,7 @@ static int tcg_reg_alloc_call(TCGContext *s, const TCGOpDef *def,
     flags = args[nb_oargs + nb_iargs + 1];
 
     nb_regs = ARRAY_SIZE(tcg_target_call_iarg_regs);
-#if TCG_TARGET_REG_BITS == 32
+#if TCG_TARGET_REG_BITS == 32 && defined(__ARM_ARCH)
```

Comment:
can you explain why this is only for ARM?

Reply:
I didn't look into the bug, but doing it for all None-VS x86 as in  https://github.com/unicorn-engine/unicorn/pull/1205/commits/15d941e847f2cdf714209bc4238083cf6d0bc1b5 the x86 test-cases failed - so instead I played it safe.
Maybe @SmailCatLog can give some insights?

Reply:
Because I don't know the macro definition of x86

Reply:
Because I don't know the macro definition of x86

Reply:
any follow up?